packages/client-cli/test/cli-graphql-variants.test.mjs

```diff
@@ -126,15 +128,19 @@ app.listen({ port: 0 });
 
   await fs.writeFile(join(dir, 'index.ts'), toWrite)
 
-  const tsconfig = JSON.stringify({
-    extends: 'fastify-tsconfig',
-    compilerOptions: {
-      outDir: 'build',
-      target: 'es2018',
-      moduleResolution: 'NodeNext',
-      lib: ['es2018']
-    }
-  }, null, 2)
+  const tsconfig = JSON.stringify(
+    {
+      extends: 'fastify-tsconfig',
+      compilerOptions: {
+        outDir: 'build',
+        target: 'es2018',
+        moduleResolution: 'NodeNext',
+        lib: ['es2018']
+      }
+    },
+    null,
+    2
+  )
```

Comment:
Is this needed still?

Reply:
Yes, otherwise the `tsc` command below will fail.